source/backend/api/Startup.cs

```diff
@@ -241,6 +242,13 @@ public void ConfigureServices(IServiceCollection services)
                 .AddCheck("liveliness", () => HealthCheckResult.Healthy())
                 .AddSqlServer(csBuilder.ConnectionString, tags: new[] { "services" });
 
+            services.AddHealthChecks()
```

Comment:
would be good to test this out in dev, but for that we may need logging as we are unlikely to be able to change the collation on the db, any ideas @eddherrera ?